lib/omnibus/packagers/fastmsi.rb

```diff
@@ -0,0 +1,654 @@
+#
+# Copyright 2015 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'pathname'
+
+module Omnibus
+  class Packager::FastMSI < Packager::Base
+    DEFAULT_TIMESTAMP_SERVERS = ['http://timestamp.digicert.com',
+                                 'http://timestamp.verisign.com/scripts/timestamp.dll']
+    id :fastmsi
+
+    setup do
+      # Render the localization
+      write_localization_file
+
+      # Render the msi parameters
+      write_parameters_file
+
+      # Render the source file
+      write_source_file
+
+      # Render the custom action file
+      write_custom_action_file
+
+      # Optionally, render the bundle file
+      write_bundle_file if bundle_msi
+
+      # Copy all the staging assets from vendored Omnibus into the resources
+      # directory.
+      create_directory("#{resources_dir}/assets")
+      FileSyncer.glob("#{Omnibus.source_root}/resources/#{id}/assets/*").each do |file|
+        copy_file(file, "#{resources_dir}/assets/#{File.basename(file)}")
+      end
+
+      # Copy all assets in the user's project directory - this may overwrite
+      # files copied in the previous step, but that's okay :)
+      FileSyncer.glob("#{resources_path}/assets/*").each do |file|
+        copy_file(file, "#{resources_dir}/assets/#{File.basename(file)}")
+      end
+
+      # Copy externally built custom action files into the staging directory
+      FileSyncer.glob("#{resources_path}/ext/*").each do |file|
+        copy_file(file, "#{staging_dir}")
+      end
+    end
+
+    build do
+      # Zip up the contents of the install directory
+      zip_command = <<-EOH.split.join(' ').squeeze(' ').strip
+        7z a -r
+          #{windows_safe_path(staging_dir)}\\#{project.name}.zip
+          #{windows_safe_path(project.install_dir)}\\*
+      EOH
+      shellout!(zip_command, returns: [0])
+
+      # Harvest the zip file with heat.exe
+      Dir.chdir(staging_dir) do
```

Comment:
Yeah I considered removing it but left it in for now just in the off chance we may need to add some other files (unknown unknowns) as we refine the install.  Will remove as part of refactoring (separate card) if we don't need it by then.